### DISCUSSION TITLE
Multi-preset conversions

### DISCUSSION THREAD

**Discussion by mkjustuk** [jlesage/docker-handbrake#300] [Q&A] [answered]
I'd like to be able to run a single source file and get three outputs based on three different presets.

Can I make this work by specifying the same mount point as watch, watch2 and watch3? Will there be any conflicts if all three 'see' the input file at the exact same time?

**Comment by jlesage** [comment]
As long as you don't configure the removal of the source file, I think it should work.

**Reply by mkjustuk** [reply, marked as answer]
Thank you, I went ahead and configured it and indeed it works perfectly.

I set watch and watch 2/3 to the same mount and then set the preset and preset_2/3 env variables as the presets I wanted, plus added three output locations to match.
